frontend/public/components/monitoring/dashboards/error.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { Alert } from '@patternfly/react-core';
+
+const ErrorAlert: React.FC<Props> = ({ message, title = 'An error occurred' }) => (
```

Comment:
I'd move this to shared components so it's a general utility we can use elsewhere.

Reply:
👍 Moved to `frontend/packages/console-shared/src/components/alerts/error.tsx `